Vizzuality/heco-invest
# Let 818 fe update the static pages footer with critical information and links

This PR updates the footer according to design and adds the coming-soon link to the pages that doesn’t exist yet

UPDATE - added privacy policy page with coming soon message

## Testing instructions

Go to any static page and you should see the updated footer [design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A3594) 

## Tracking

[LET-818](https://vizzuality.atlassian.net/browse/LET-818)
